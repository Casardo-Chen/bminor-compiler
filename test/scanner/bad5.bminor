// basic data types


character:	 	char = 'c';
character: char = '\n';
character = '\6';
character = '\c'
character = '\t'