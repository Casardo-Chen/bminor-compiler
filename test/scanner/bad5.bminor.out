IDENTIFIER character
COLON :
CHAR char
ASSIGN =
CHARACTER_LITERAL c
SEMICOLON ;
IDENTIFIER character
COLON :
CHAR char
ASSIGN =
CHARACTER_LITERAL 

SEMICOLON ;
IDENTIFIER character
ASSIGN =
encode error: Invalid escape sequence
scan error: "\6" is not a valid character
